drivers/couchbase/README.md

```diff
@@ -0,0 +1 @@
+# Parent project for all the community Couchbase drivers
```

Comment:
@osantana85 can you elaborate what should be done here ?
Just add more description into the README.md file? Or I can just share the link to the official docs?

Reply:
Hello @babltiga, sorry for the confusion. This comment is here because you need to add the dependency to a file that was not modified by this PR (mongock-community-bom > pom.xml). You don't need to modify the README.md file.